java/test/jmri/jmrit/signalsystemeditor/configurexml/LoadAndStoreAllSignalSystemsTest.java

```diff
@@ -359,23 +359,26 @@ private void loadAndStoreFileCheck(File file) throws Exception {
                 signalMastXml.save(signalSystem, signalMastType, true);
             }
 
+            File parentFile = file.getParentFile();
+            Assertions.assertNotNull(parentFile);
             File compFile = new File(FileUtil.getProfilePath()
-                    + "xml/signals/" + "/" + file.getParentFile().getName() + "/" + file.getName() );
+                    + "xml/signals/" + "/" + parentFile.getName() + "/" + file.getName() );
             checkFile(compFile, file);
         }
     }
 
     @ParameterizedTest(name = "{index}: {0} (pass={1})")
     @MethodSource("data")
     public void loadAndStoreTest(File file, boolean pass) throws Exception {
-        if (!file.getParent().equals("xml/signals") && !file.getParent().equals("xml\\signals")) {
+        String parentFile = file.getParent();
+        Assertions.assertNotNull(parentFile);
+        if (!parentFile.equals("xml/signals") && !parentFile.equals("xml\\signals")) {
             loadAndStoreFileCheck(file);
         }
     }
 
     @BeforeEach
     public void setUp(@TempDir File tempDir) throws IOException  {
-        tempDir = new File("temp/temp/SignalSystemEditor");
```

Comment:
Please comment this line instead of removing it. The reason I did this is that the temporary files are removed before the test ends. So it's impossible to see the temporary files after the test is run. Sometimes it's needed to compare the temporary files with the original files to see the differences.